Tools/autotest/rover.py

```diff
@@ -4682,9 +4691,11 @@ def test_wheelencoders(self):
         '''make sure wheel encoders are generally working'''
         ex = None
         try:
-            self.set_parameter("WENC_TYPE", 10)
-            self.set_parameter("EK3_ENABLE", 1)
-            self.set_parameter("AHRS_EKF_TYPE", 3)
+            self.set_parameters({
+                "WENC_TYPE": 10,
+                "EK3_ENABLE": 1,
```

Comment:
We use EKF3 by default now so I don't think we should need these two parameter setting do we?  maybe I'm wrong.

Reply:
I think technically you should be able to run the test suite with `./Tools/autotest/autotest.py build.Rover test.Rover --force-ahrs-type 2` to run the Rover test suite under EKF2.  Since this test would fail under those circumstances, we need to force EKF3 here, I think.